salt/metalk8s/orchestrate/deploy_node.sls

```diff
@@ -207,6 +207,41 @@ Check pillar before highstate:
       - salt: Sync module on the node
       - http: Wait for API server to be available before highstate
 
+
+{#- Nodes in 2.6 or lower rely on manual provisioning of loop devices, which
+    we need to clean-up to avoid having duplicates generated by the systemd
+    units, introduced in 2.7 #}
+{%- if pillar.orchestrate.get('cleanup_loop_devices', False) %}
+  {%- if pillar.orchestrate.get('skip_draining', False) %}
+
+Cannot cleanup devices if skipping drain:
+  test.fail_without_changes: []
+
+  {%- else %}
+    {%- set loop_volume_ids = [] %}
+    {%- for volume in pillar.metalk8s.volumes %}
```

Comment:
Ah, that can't work, this pillar is computed for a minion in particular... I guess the state should handle it on its own.